tokio-postgres/tests/test/main.rs

```diff
@@ -304,6 +304,7 @@ async fn cancel_query_raw() {
     let client = connect("user=postgres").await;
 
     let socket = TcpStream::connect("127.0.0.1:5433").await.unwrap();
+    #[allow(deprecated)]
     let cancel = client.cancel_query_raw(socket, NoTls);
```

Comment:
Let's just switch these tests over to use the new APIs.

Reply:
You bet. Done.